masterbase/steam.py

```diff
@@ -354,12 +359,31 @@ def query(self) -> list[Server]:
         return servers
 
 
-def is_limited_account(steam_id: str) -> bool:
-    """Determine if an account is limited or not."""
+@cache
+def player_summary(steam_id: str) -> dict[str, str | int] | None:
+    """Retrieve a player summary."""
     url = "https://api.steampowered.com/ISteamUser/GetPlayerSummaries/v2"
     params = {"key": get_steam_api_key(), "steamids": steam_id}
     response = requests.get(url, params).json()
-    player_data = response["response"]["players"][0]
-    limited = not bool(player_data.get("profilestate", False))
+    if len(response["response"]["players"]) > 0:
+        return response["response"]["players"][0]
+    else:
+        return None
```

Comment:
Maybe cleaner to have this just return an empty dict instead of None? That cleans up the return annotation as well as the truthy check in both limited_account and account_exists

Reply:
addressed in https://github.com/MegaAntiCheat/masterbase/pull/58/commits/79e7714b90e0d721eaa571c8e8a11519fa046ce9